nidaqmx/_grpc_stub_interpreter.py

```diff
@@ -0,0 +1,394 @@
+import threading
+import warnings
+
+import grpc
+import nidaqmx.nidaqmx_pb2 as grpc_types
+import nidaqmx.nidaqmx_pb2_grpc as nidaqmx_grpc
+import nidaqmx.session_pb2 as session_grpc_types
+
+class GrpcStubInterpreter(object):
+    '''Interpreter for interacting with a gRPC Stub class'''
+
+    def __init__(self, grpc_options):
+        self._grpc_options = grpc_options
+        self._lock = threading.RLock()
+        self._client = nidaqmx_grpc.NiDAQmxStub(grpc_options.grpc_channel)
+        self.session_initialization_behavior = grpc_options.initialization_behavior
+        self.set_task_handle()
+        self.set_system_handle()
+        self.set_scale_handle()
+
+    def set_task_handle(self, value=session_grpc_types.Session()):
+        self._task_handle = value
+
+    def get_task_handle(self):
+        return self._task_handle
```

Comment:
This means that each task will have a separate interpreter (like nimi-python). I guess that's okay, as long as system/scale objects can use an interpreter that does not have a task handle.